ert_shared/models/multiple_data_assimilation.py

```diff
@@ -254,11 +254,10 @@ def create_context(
         if initialize_mask_from_arguments:
             mask = self._simulation_arguments["active_realizations"]
         else:
-            state = (
+            mask = sim_fs.getStateMap().createMask(
```

Comment:
You've removed intermediate `state` here, but not in EnsembleSmoother i.e. `ert_shared/models/ensemble_smoother.py` :)

Reply:
fixed!